sstables/sstable_directory.cc

```diff
@@ -56,6 +57,20 @@ sstable_directory::system_keyspace_components_lister::system_keyspace_components
 {
 }
 
+static std::unique_ptr<sstable_directory::components_lister>
```

Comment:
Better to make it non-static and use `this->_storage_opts` and others rather than passing them as arguments

Reply:
great. even "closer" ! in the sense that this function is a member of this class now! =)